### PR TITLE
Updated back arrow to navigate to correct page

### LINKED ISSUE #1478: bug: Back arrow navigates to wrong page
### Description

If you search for a profile from the search page, choose a profile page then click the arrow icon in the menu on the profile page to back it takes you directly to the home page and not back to the search page.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by andrewkykoo**
## Fixes Issue

Closes #1478 

## Changes proposed

- Updated the Link component's to prop to '/search' in Navbar component in User component

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

https://user-images.githubusercontent.com/47183818/178357822-ff887d86-84f2-4117-9f43-277f29b1d1fe.mov

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1482"></a>

### CODE REVIEW

File: src/Components/UserProfile/User.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ function User({ singleUser }) {
       {!singleUser && (
         <Navbar
           start={
-            <Link to="/" aria-label="Go back to Home">
+            <Link to="/search" aria-label="Go back to Home">
```
Comment (kumarsonsoff3) [inline src/Components/UserProfile/User.js:41]: You should have to change the label too! 
```suggestion
            <Link to="/search" aria-label="Go back to Search">
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please **Revert** the changes from the `package-lock.json` file! Let us know If you need any help!

File: src/Components/UserProfile/User.js (hunk 1 above)
Comment (andrewkykoo) [reply src/Components/UserProfile/User.js]: Got it! Thanks for the feedback 👍

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM! 👍 Thanks for contributing! 🚀


✅  **Summary:** Code looks Good.  | **Tags:** Maintainable

**Review by emmalearnscode — APPROVED** [review]
Brilliant! Thanks for your contribution!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**